### PR TITLE
Make cookies last longer

### LINKED ISSUE #218: Use longer lasting cookie
Currently when you quit your browser, the login session is not persisted. It would be cool if it was persisted for some period of time to avoid having to re-login frequently.

### PR DESCRIPTION
**Pull request by stephencelis**
Fixes #218.

### CODE REVIEW

File: Sources/PointFreeTestSupport/PointFreeTestSupport.swift [hunk 1]
```diff
@@ -24,7 +24,7 @@ extension Environment {
   public static let teamYearly = mock
     |> \.database.fetchSubscriptionTeammatesByOwnerId .~ const(pure([.mock]))
     |> \.database.fetchTeamInvites .~ const(pure([.mock]))
-    |> \.stripe.fetchSubscription .~ const(pure(.teamYearly))
+    |> (\Environment.stripe.fetchSubscription) .~ const(pure(.teamYearly))
```
Comment (stephencelis) [inline Sources/PointFreeTestSupport/PointFreeTestSupport.swift]: Heh, this snuck in because it sped up compile time here quite a bit.

**Comment by stephencelis** [comment]
Tested locally and my session now persists across browser sessions.

**Comment by stephencelis** [comment]
Happy now @mergesort?

File: Sources/PointFree/Session.swift [hunk 2]
```diff
@@ -14,7 +14,10 @@ public func writeSessionCookieMiddleware<A>(_ update: @escaping (Session) -> Ses
       return setCookie(
           key: pointFreeUserSession,
           value: value,
-          options: [.path("/")]
+          options: [
+            .expires(AppEnvironment.current.date().addingTimeInterval(60 * 60 * 24 * 365 * 10)),
```
Comment (mbrandonw) [inline Sources/PointFree/Session.swift]: would we even do `Date.distantFuture`?

File: Sources/PointFreeTestSupport/PointFreeTestSupport.swift (hunk 1 above)
Comment (mbrandonw) [reply Sources/PointFreeTestSupport/PointFreeTestSupport.swift]: ha I bet

File: Sources/PointFree/Session.swift (hunk 2 above)
Comment (stephencelis) [reply Sources/PointFree/Session.swift]: Ha, that's the year 4,000. A little optimistic, are we?

My only worry is some browsers handling this stuff weird and breaking cookies for some folks.

**Comment by mergesort** [comment]
>Happy now @mergesort?

This seems a bit overly philosophical to answer in a PR.

**Pull request merged**